examples/ElmFrontendRocBackend/backend.roc

```diff
@@ -5,15 +5,20 @@ import pf.Task exposing [Task]
 import pf.Http exposing [Request, Response]
 import pf.Utc
 
+# [backend](https://chatgpt.com/share/7ac35a32-dab5-46d0-bb17-9d584469556f) Roc server
```

Comment:
Did you mean to leave this here?

Reply:
Yes, it's intended for those who don't know what a backend server is